versioneer.py

```diff
@@ -1452,7 +1451,7 @@ def get_versions(verbose=False):
     try:
         ver = versions_from_file(versionfile_abs)
         if verbose:
-            print("got version from file %s %s" % (versionfile_abs, ver))
+            print("got version from file {} {}".format(versionfile_abs, ver))
```

Comment:
f-string this

Reply:
Are you reading for these, or do you have a tool finding them?

Reply:
I read through all of the changes to find the ones the upgrade script missed.